srcpkgs/ibus-rime/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'ibus-rime'
+pkgname=ibus-rime
+version=1.3.0
+revision=3
+build_style=gnu-makefile
+make_cmd=make
+hostmakedepends="pkg-config cmake"
```

Comment:
why cmake ? if it's using cmake then it should use the cmake build_style=

Reply:
this package uses a strange build prcess where it uses gnu make to launch cmake which makes a makefile then makes within a make...

Reply:
@youngjinpark20 You can use the cmake build_style. It may be required to overwrite the `do_install` method with smth like but otherwise it should work fine.
```sh
do_install() {
    make PREFIX=/usr DESTDIR=${DESTDIR} install
}
```